cfgov/cfgov/settings/base.py

```diff
@@ -587,3 +588,25 @@
     # and the table will not be sortable until cf-tables from CF 4.x is implemented
     'SORTABLE_TABLES': {},
 }
+
+
+# Watchman tokens, used to authenticate global status endpoint
+WATCHMAN_TOKENS = os.environ.get('WATCHMAN_TOKENS', os.urandom(32))
```

Comment:
Because the default is `os.urandom(32)`, the status endpoint will never be available (unless you can guess the random string) on servers where `WATCHMAN_TOKENS` is not in the environment.